py3status/modules/file_status.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display if a file or dir exists.
+Display status for file or directory.
```

Comment:
I think the original `Display if a file or dir exists.` was a better description  `status` implies last modified, permissions etc imho